### DISCUSSION TITLE
blog/hands-on-with-envoy-gateway/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#86] [Announcements]
# blog/hands-on-with-envoy-gateway/

最近 Envoy Gateway 0.2 发布了，API 网关的生态系统迎来了新的变化。这篇文章将想你介绍 Kubernetes API 网关领域的最新进展。

https://lib.jimmysong.io/blog/hands-on-with-envoy-gateway/

**Comment by hardy4yooz** [comment]
`envoy-default-insecure-port-5879556bd4-r5ggw` 应该是在 `Gateway`创建的时候被创建的。

**Reply by rootsongjc** [reply]
是的，没错。

**Reply by hardy4yooz** [reply]
嗯，文章里我第一反应以为是创建 `HTTPRoute` 的时候。
